src/js/components/DropButton/stories/Menu.js

```diff
@@ -44,6 +43,7 @@ const MenuDropButton = () => {
   );
 };
 
-storiesOf('DropButton', module).add('Menu', () => <MenuDropButton />, {
+export const Menu = () => <MenuDropButton />;
+Menu.parameters = {
```

Comment:
Hi @leossantiago, Here is the correct syntax to disable chromatic on a story:
```
Menu.story = {
  parameters: {
    chromatic: { disable: true },
  },
};
```

Reply:
Thanks! I'll fix that :heart: